node/cn/gasprice/feehistory.go

```diff
@@ -101,8 +108,16 @@ func (oracle *Oracle) processBlock(bf *blockFees, percentiles []float64) {
 		bf.results.nextBaseFee = new(big.Int).SetUint64(params.ZeroBaseFee)
 	}
 
-	// There is no GasLimit in Klaytn, so it is enough to use pre-defined constant in api package as now.
-	bf.results.gasUsedRatio = float64(bf.header.GasUsed) / float64(params.UpperGasLimit)
+	// Use gasTarget as gasLimit for gasUsedRatio.
+	if chainconfig.IsMagmaForkEnabled(big.NewInt(int64(bf.blockNumber))) {
+		// TODO-Klaytn: Instead of using chainConfig, we should use governance set values.
+		if chainconfig.Governance.KIP71.GasTarget != 0 {
+			bf.results.gasUsedRatio = float64(bf.header.GasUsed) / float64(chainconfig.Governance.KIP71.GasTarget)
```

Comment:
1. Please divide by MAX_BLOCK_GAS_USED_FOR_BASE_FEE as per https://github.com/blukat29/kips/blob/priority-fee-mechanism/KIPs/kip-162.md.
2. If the ratio exceeds 1, then set it to 1.

Reply:
Done, thanks! b96ff95